assets/js/blocks-registry/payment-methods/can-make-payment-extensions.js

```diff
@@ -0,0 +1,7 @@
+// Keeps callbacks registered by extensions for different payment methods
```

Comment:
Assuming we want this config object to be something we can add additional functionality to in the future, it might be good to name the file something like `extensions-config.js` instead.

Also, maybe do this in TS from the get-go?